Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build the manager binary
-FROM golang:1.18 as builder
+FROM golang:1.19 as builder
+ARG TARGETOS
+ARG TARGETARCH
```

Comment:
THis might need a change in `build-push.sh`.

Reply:
Hmm actually it's not needed to introduce any changes to the `build-push.sh` shell script. It does work the same way it did as before. And if we want we could now make it possible to set the OS and ARCH. 

AFAIK there does not exist a pipeline to build the fastly controller but I might be wrong. Maybe  @shreddedbacon might know more. Within the pipeline we should actually use `make docker-buildx` and specify `PLATFORM` or just use another way to build the image for example using `kaniko` which is used for the `ingress-operator`. 

Maybe we should just remove the `build-push.sh` script as all it does is possible to do by using `make docker-buildx`.

Reply:
`build-push.sh` was only used for locally building and pushing images.

I'm happy with whatever you plan to do here now, I'll classify myself as no longer a maintainer of this tool.